packages/release-utils/lib/releases.js

```diff
@@ -0,0 +1,173 @@
+const fs = require('fs');
+const os = require('os');
+const Promise = require('bluebird');
+const requestPromise = require('request-promise');
+const request = require('request');
+
+const localUtils = require('./utils');
+
+module.exports.draft = (options = {}) => {
+    let draft = true;
+    let prerelease = false;
+
+    if (!options.changelogPath) {
```

Comment:
We should maybe make a little helper for config like
```js
const get = o => p => p.split('.').reduce((obj, prop) => obj && obj[prop], o);

const getMissingOptions = (obj, ...options) => {
    return options.filter(get(obj));
}
```
usage:
```js
module.exports.draft = (options = {}) => {
    let draft = true;
    let prerelease = false;

    const missingOptions = getMissingOptions(options,
        'changelogPath',
        'github',
        'github.username'
        // etc...
    );

    if (missingOptions.length) {
        throw new Error(`${missingOptions[0]} is required`);
        // or
        throw new Error(`Missing options: ${missingOptions.join(',')}`);
    }
```

Reply:
random though :shrimp:

Reply:
Perfect yes 👍